app/server/middleware/session.ts

```diff
@@ -18,6 +19,21 @@ export function createSession(useRedisStore = false): RequestHandler {
     `Using redis store with secure cooke: ${secure} secret length: ${secret?.length}`
   );
 
+  if (isFeatureEnabled(Feature.HTTPONLY_COOKIE_FLAG_ENABLED)) {
+    return session({
+      cookie: {
+        httpOnly: true,
+        sameSite: true,
+        maxAge: config.get('session.cookie.maxAgeInMs'),
+        secure,
+      },
+      resave: true,
+      saveUninitialized: true,
+      secret,
+      rolling: true,
+      store,
+    });
+  }
```

Comment:
a bit confused here, isn't the same code repeated below?
same observation as above in terms of feature flag use